desktop/apps/partnerships/templates/nav.jade

```diff
@@ -5,7 +5,7 @@
     for section, i in sections.slice(0, -1)
       a.partnerships-nav-link.internal( href="/#{subject}-partnerships/#{section.slug}", data-section=section.title )
         span= section.title
-    a.partnerships-nav-link.partnerships-nav-apply-link.avant-garde-button-black.is-small.apply-link-cta(
+    a.internal.partnerships-nav-link.partnerships-nav-apply-link.avant-garde-button-black.is-small.apply-link-cta(
```

Comment:
This `.internal` class is what indicated scrolling down the page. I think at one point this linked out to a separate Marketo page, but from the Slack conversation it appears we want this to scroll down the page again—so adding it back in.